package ExceptionHandling;

public class ControlFlow {

	public static void main(String[] args) {
		
		try {
			
			System.out.println("stmt-1");
			System.out.println("stmt-2");
			System.out.println("stmt-3");
		}
		catch(ArithmeticException e) {
			System.out.println("stmt-4");	
		}
		finally {
			System.out.println("stmt-5");
		}
		
		System.out.println("stmt-6");
	}

}
